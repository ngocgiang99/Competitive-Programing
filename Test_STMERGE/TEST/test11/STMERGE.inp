10
1 9
272910324 920530219 239964938 108813497 681652150 250999148 912815368 948309384 303156244 
4 9
21483477 357883589 999877345 361937190 920239623 678879676 123034846 150739761 370482881 
841684784 950645070 877496554 438857960 185301908 791782710 411273959 189783970 806163358 
491463131 559906299 68125780 713279043 582904003 93912894 5059747 315668664 504880050 
449451011 347248755 573999218 489395372 952923103 771859945 302410534 504109560 674431143 
6 10
703311898 680417905 63952799 998083471 185004504 519037841 779908968 825233902 575069237 630339741 
416192491 734709915 650367096 6512745 651931790 412600459 450403876 381913611 506005237 367247603 
135320320 13191382 132743943 946076488 696576784 467570786 148873446 178386751 854350489 49104513 
668787962 21037773 483638304 630534474 28519398 773624358 487208555 292181778 508866704 759374785 
900232829 995314797 755714588 136002641 272724893 728200752 838821360 390605005 119723342 799753584 
379364273 119935080 395454807 678783363 636962628 359473816 588074940 69038078 59728589 321705234 
3 2
477702321 318931403 
411237615 812416957 
458986902 79401207 
6 9
279767649 962341645 753674766 166728789 137146423 122054848 548478795 24387919 784850044 
439927559 632760898 477577071 408707286 253413653 530788536 331361801 47111977 174429792 
577909938 354958515 404295068 187741341 700299381 839168321 68072593 596303297 858034311 
206038984 479666352 220057089 373672363 417648286 926238428 987101138 935102184 135433815 
607841338 232300521 723166594 78556104 951555633 506655034 828499710 873098247 332822116 
29867259 987086113 442239080 835542712 930411170 644918768 946782337 900296532 898759650 
1 6
655767195 255949489 684010215 963268972 496532786 300073796 
9 9
421857791 784617488 209592587 21355633 460690983 826978250 284582978 839065498 831991054 
404420966 187137411 854840 623763035 912268315 413158547 226081440 316045296 293502300 
904840056 481353576 381326837 39603936 658308213 515286372 526179021 654761626 316783633 
172419241 219250973 528826560 852492048 879417229 579122636 2597831 352507511 18458277 
595532285 326601316 786896779 520516804 91235093 992037771 940777425 637675777 360760506 
474647667 339423503 736571569 778857169 357339280 624704840 7741288 182076799 998877590 
552367811 76428539 639321384 2609531 539545604 705026432 204626546 295677045 184002000 
957587787 722636353 829484279 767714092 394151153 46087611 260450743 206192477 180103496 
383389765 310332842 928206313 352235364 358998403 63711236 340917395 297570971 200145755 
4 2
236662401 606414834 
39326214 268421274 
593817323 755046316 
607086934 800945856 
5 7
928921186 366349977 330354145 931563872 900176800 616162602 44513111 
10475363 371039683 724484536 562942757 233581726 57167809 184020425 
282902112 642078901 239924431 533873441 505987678 645641614 243989602 
629969348 343572659 221233998 510375470 497771513 762513716 197924441 
15061034 546814532 926202359 89473877 796361865 431314283 479545169 
7 8
42550897 685669333 227548915 5074056 829600412 458999898 902759967 740330449 
13835634 868980213 57095760 553599698 567069879 96851450 46827882 198612224 
978727536 765187582 161063827 109886992 282513634 976708567 643154735 310014967 
180986002 820289433 775054053 55125737 154223685 311294361 172614551 172445533 
761998593 508698733 446249069 393520646 563965962 444877033 842276554 165857242 
182893284 665611299 405700180 979452430 672544598 765288815 142426863 798188056 
454386532 249344098 3643073 643897258 675160916 900344018 987887151 574937276 
