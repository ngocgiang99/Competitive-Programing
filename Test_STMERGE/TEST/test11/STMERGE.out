272910324
368732232
119570057
318931403
29867259
300073796
267654800
505083675
15061034
241163121
